hvplot/__init__.py

```diff
@@ -209,6 +209,54 @@ def _hook_patch_docstrings(backend):
 
 @_wraps(_pn.bind)
```

Comment:
Doesn't adding a docstring mean that the usage of `wraps` becomes useless? If it's not removed, does it mean that `bind` defined here is still going to inherit from the docstring of `pn.bind`? I'm not quite sure as how this behaves.

Reply:
I think you are right, but will test.

Wraps does more than adding a docstring dynamically.

Pyright and VS Code has a hack for static typing to also use the docstring and function signature.